devops/scripts/set_contributor_sp_secrets.sh

```diff
@@ -9,14 +9,14 @@ set -e
 #
 # Required environment variables:
 #
-#   - TF_VAR_tre_id - The TRE ID, used to deduce the Key Vault name
-#   - SUB_ID - The Azure subscription ID
+#   - TRE_ID - The TRE ID, used to deduce the Key Vault name
+#   - ARM_SUBSCRIPTION_ID - The Azure subscription ID
```

Comment:
Here you are removing SUB_ID and adding ARM_SUBSCRIPTION_ID. In the .env.sample SUB_ID is used

Reply:
Sorry, it is fixed now.